src/settings-ui/Settings.UI/SettingsXAML/Flyout/LaunchPage.xaml

```diff
@@ -107,7 +107,11 @@
                 x:Uid="UpdateAvailable"
                 IsClosable="False"
                 IsOpen="{x:Bind ViewModel.IsUpdateAvailable, Mode=OneWay}"
-                Severity="Success" />
+                Severity="Informational">
+                <InfoBar.ActionButton>
+                    <HyperlinkButton x:Uid="ShowUpdateDetails" Click="UpdateDetailsBtn_Click" />
```

Comment:
What does this look like in e.g. German? I can imagine the icons would overlap with this text?

Reply:
Yeah, so that's my point.. wouldn't that overlap with the icons on the right? Also in other languages where text is longer.

If so, we might want to hold out of adding this in the flyout.. and maybe replace the `InfoBar` with a button / tooltip all together?

Reply:
The best solution would be a custom control with a info badge, blue background and only a link "Update available".

Reply:
I thought on:
- Blue stackpanel background.
- No button.
- Instead of text element an hyperlink element.

But I can try on weekend. Do you know the resource name of the blue infobar background?
And the corner radius of the infobar?

Reply:
```xml
<StackPanel Orientation="Horizontal" Background="{ThemeResource InfoBarInformationalSeverityBackgroundBrush}">
  <InfoBadge/>
  <HyperlinkButton Content="Update available"/>
</StackPanel>

Reply:
@niels9001
1. Using the stackpanel-hyperlink-version in the dashboard page crashes settings. Do you know why.
2. Using the stackpanel-hyperlink-version the flyout works. But the infobadge is vertical shrunk. ![image](https://github.com/microsoft/PowerToys/assets/61519853/cbebbbc5-9304-44d5-b0d2-8d6c717ff91c)